README.md

```diff
@@ -54,3 +54,59 @@ Cloudflare Client V4 API Email of the current user.
 to each request.
 
 5. Build an API Client for the Cloudflare Host API which adds the Host Key to all requests.
+
+## config.json 
+
+config.json has the following structure:
+
+```
+{
+    "debug": false,
+    "featureManagerIsFullZoneProvisioningEnabled": false,
+    "isDNSPageEnabled": true,
+    "isSubdomainCheckEnabled": true,
+    "homePageCards": [
+        "ApplyDefaultSettingsCard",
+        "AutomaticHTTPSRewritesCard",
+        "IpRewriteCard",
+        "PluginSpecificCacheCard",
+        "PluginSpecificCacheTagCard"
+    ],
+    "moreSettingsCards": {
+        "container.moresettings.speed": [
```

Comment:
Do we want to keep the cards? Since we don't explicitly want people to configure them?